svelte-kit-scss/src/lib/components/shared/Dropdown/DropdownMenu.svelte

```diff
@@ -12,14 +13,21 @@
   };
 
   $: descriptionDisplay = description || String();
+  type Direction = 'left' | 'right';
+  export let direction: Direction = 'right';
```

Comment:
As a convention its a good idea to have all the "props" together near the top of the script